assets/js/admin/wc-enhanced-select.js

```diff
@@ -147,7 +147,7 @@ jQuery( function( $ ) {
 
 				$( this ).select2( select2_args ).addClass( 'enhanced' );
 
-				if( $( this ).data( 'sortable' ) == true ){
+				if( $( this ).data( 'sortable' ) === true ){
```

Comment:
There should be a space between `if` and the `parenthesis`. We should also use yoda style here. Ditto with line 230 down below.